layouts/partials/continuous_integration/ci-tests-setup.html

```diff
@@ -0,0 +1,56 @@
+{{ $dot := . }}
+<div class="ci-cd-tests">
+  <div class="container cards-dd col-num-2">
+    <div class="row row-cols-2 row-cols-sm-4 g-2 g-xl-3 justify-content-sm-center">
+      <div class="col">
+        <a class="card h-100" href="/continuous_integration/tests/setup/dotnet/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/dotnet_avatar.svg" "class" "img-fluid" "alt" ".net" "width" "200") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_integration/tests/setup/java/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/java_avatar.svg" "class" "img-fluid" "alt" "java" "width" "200") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_integration/tests/setup/javascript/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/javascript.png" "class" "img-fluid" "alt" "javascript" "width" "200") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_integration/tests/setup/python/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/python_avatar.svg" "class" "img-fluid" "alt" "python" "width" "200") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_integration/tests/setup/ruby/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/ruby_avatar.svg" "class" "img-fluid" "alt" "ruby" "width" "200") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_integration/tests/setup/swift/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/swift_avatar.svg" "class" "img-fluid" "alt" "swift" "width" "200") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_integration/tests/setup/junit_xml/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/junit.png" "class" "img-fluid" "alt" "upload junit tests" "width" "200") }}
```

Comment:
This is likely going to be confused by the java test framework, which is not quite what we want: many test frameworks for languages different than Java can create junit xml reports, but that's not the framework. We had a similar discussion when doing the onboarding page and we left the JUnit XML option without a logo: https://app.datadoghq.com/ci/setup/test?language=junit-xml